Header header
uint32 vhcid
geographic_msgs/GeoPoint geo
